package.json

```
@@ -43,6 +46,8 @@
 		"ts": "tsc -b",
 		"build": "tsdown",
 		"prepublish": "bun run build",
-		"prepare": "husky"
+		"prepare": "husky",
+		"dev:sse": "tsx src/server-sse.ts",
+		"start:sse": "node --loader tsx src/server-sse.ts"
```

Not a deal breaker, but you could just replace `tsx` with `bun` here and get rid of a dependency:


```suggestion
		"dev:sse": "bun run src/server-sse.ts",
		"start:sse": "bun run src/server-sse.ts"
```